features/order/managing_orders/seeing_order_aggregated_taxes.feature

```diff
@@ -0,0 +1,47 @@
+@managing_orders
+Feature: Seeing aggregated taxes of an order
+    In order to be aware of taxes in an order
+    As an Administrator
+    I want to see aggregated taxes of a specific order
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And there is a zone "EU" containing all members of the European Union
+        And default tax zone is "EU"
+        And the store has "EU VAT" tax rate of 23% for "Quantum mechanics" within "EU" zone
+        And the store has "EU Low VAT" tax rate of 10% for "Theoretical physics" within "EU" zone
+        And the store has a product "Schrödinger's cat" priced at "€100.00"
+        And it belongs to "Quantum mechanics" tax category
+        And the store has a product "General relativity" priced at "€50.00"
+        And it belongs to "Theoretical physics" tax category
```

Comment:
I enjoy these "Easter Eggs" too, but they make sense for irrelevant info, like customer name. Here we should use something that will be easy to understand and as close as possible to a real world scenario.